modules/openshift-developer-cli-deploying-the-front-end-component.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/openshift_developer_cli/creating-a-multicomponent-application-with-odo.adoc
+// * cli_reference/openshift_developer_cli/creating-an-application-with-a-database.adoc
+
+[id="deploying-the-front-end-component_{context}"]
+
+= Deploying the front-end component
+
+.Procedure
+
+. Change the current directory to the front-end directory:
++
+----
+$ cd <directory-name>
+---- 
+
+. List the contents of the directory to see that the front end is a Node.js application.
++
+----
+$ ls
+assets  bin  index.html  kwww-frontend.iml  package.json  package-lock.json  playfield.png  README.md  server.js
+---- 
++
+[NOTE]
+====
+The front-end component is written in an interpreted language (Node.js); it does not need to be built.
+====
+
+. Create a component configuration of Node.js component-type named `frontend`:
++
+----
+$ odo create nodejs frontend
+ ✓  Validating component [5ms]
+Please use `odo push` command to create the component with source deployed
+----
+
+ifeval::["{context}" == "creating-an-application-with-a-database"]
+. Create a URL to access the frontend interface.
++
+----
+$ odo url create myurl
+ ✓  URL myurl created for component: nodejs-nodejs-ex-pmdp
+----
+
+. Push the component to the {product-title} cluster.
++
+----
+$ odo push
+Validation
+ ✓  Checking component [7ms]
+
+
```

Comment:
To much spacing... Should only be one line..

Reply:
Fixed.